package.json

```diff
@@ -47,14 +48,16 @@
     "svelte": "4.2.19",
     "svelte-check": "4.0.2",
     "svelte-preprocess": "5.1.4",
-    "tailwindcss": "3.4.11",
+    "tailwindcss": "3.4.13",
     "tweetnacl": "1.0.3",
     "typescript": "5.6.2",
     "vite": "5.4.6"
   },
   "devDependencies": {
     "@sveltejs/vite-plugin-svelte": "3.1.2",
-    "@tsconfig/svelte": "5.0.4"
+    "@tsconfig/svelte": "5.0.4",
+    "msw": "2.3.1",
```

Comment:
These were already previously audited - no concerns with the additional dependencies.